tests/common/test_lab.py

```diff
@@ -39,9 +39,9 @@ def add_member_team(common_lab, add_admin):
                 last_name="Basic",
             ),
             dict(
-                lab_member_name="This Loner",
+                lab_member_name="This Loner",  # codespell:ignore loner
```

Comment:
I don't think this value is ever read, so I think I'd just change this to an accepted word. Maybe `Solo`? Low prioroty